toolchain/build-xenon-toolchain

```diff
@@ -303,7 +303,7 @@ function filesystems_install
 {
 	echo -e "Building Filesystems..."
 
-	filesystems="fat-xenon ext2fs-xenon xtaflib" # ntfs-xenon ext2fs-xenon xtaflib
+	filesystems="fat-xenon ext2fs-xenon" # xtaflib ntfs-xenon ext2fs-xenon xtaflib
```

Comment:
Why did you disable the xtaflib here?

Reply:
This was a mistake - I'll need to test if it works with the new toolchain first